staff/views/coupon.py

```diff
@@ -139,7 +144,27 @@ def post(self, request):
         csv_file = form.cleaned_data.get('csv_file')
         csvreader = csv.DictReader(codecs.iterdecode(csv_file, 'utf-8'))
         csv_rows = []
+        course_basics = Course.objects.get(name=settings.CODING_BASICS)
+        course_bootcamp = Course.objects.get(name=settings.CODING_BOOTCAMP)
+        coupon_effect_basics = CouponEffect.objects.filter(
+            couponable_id=course_basics.id,
+            discount_type='dollars',
+            discount_amount='20'
+        ).first()
+        coupon_effect_bootcamp = CouponEffect.objects.filter(
+            couponable_id=course_bootcamp.id,
+            discount_type='dollars',
+            discount_amount='200'
+        ).first()
+
         for row in csvreader:
+            coupon = Coupon.objects.create(
+                start_date=make_aware(datetime.datetime.now()),
+                end_date=None,
+                description=row['email']
+            )
+            coupon.effects.set([coupon_effect_basics, coupon_effect_bootcamp])
+            coupon.save()
             csv_rows.append(row)
```

Comment:
we probably don't need this anymore. instead, this view should redirect to a new ListBatchView where we query all the recently created coupons and show them in a page.

Reply:
my thinking is that since this is a `new` view, it should probably just be concerned with creating. any display should probably be in a separate view.

Reply:
will get on this